src/main/java/uk/gov/justice/hmpps/prison/api/resource/CaseNoteResource.java

```diff
@@ -46,7 +46,10 @@ public class CaseNoteResource {
 
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "The case note usage list is returned.")})
-    @Operation(summary = "Retrieves list of case notes grouped by type/sub-type and staff", description = "Retrieves list of case notes grouped by type/sub-type and staff. Requires role VIEW_CASE_NOTES")
+    @Operation(
+        deprecated = true,
+        summary = "This endpoint has been replaced in the case notes service - see case-notes-service/case-notes/staff-usage"
```

Comment:
would be good to add the replacement information to the `summary` and `description` rather than just replacing all the content esp. removing the role requirement

Reply:
I take it the standard `@Deprecated` annotation didn't then do anything in the docs?